java/vector/src/main/codegen/templates/NullableValueVectors.java

```diff
@@ -44,6 +50,7 @@
  * NB: this class is automatically generated from ${.template_name} and ValueVectorTypes.tdd using FreeMarker.
  */
 @SuppressWarnings("unused")
+@Deprecated
```

Comment:
Add "Use XXX instead" in deprecation warning?

Reply:
Sure.